db/migrate/20161129051115_drop_conversations_table.rb

```diff
@@ -0,0 +1,4 @@
+class DropConversationsTable < ActiveRecord::Migration[5.0]
```

Comment:
This migration is blank, so it doesn't do anything. Also drop table and remove column migrations need special things to be reversible. We'll want to add those into this migration when we update it.